### PR TITLE
solved all 14 questions

### PR DESCRIPTION
**Pull request by AryaChakraborty**

### CODE REVIEW

File: if-else questions/absolute.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    int n;
+    cin >> n;
+    if (n > 0)
+    {
```
Comment (Vishal-raj-1) [inline if-else questions/absolute.cpp:10]: no need of writing `std::` again and again. We have already declared using namespace std;

File: if-else questions/grade_card.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        std::cout << "C" << std::endl;
+    }
+    else if (marks >= 60 && marks <= 80)
+    {
+        std::cout << "B" << std::endl;
+    }
+    else if (marks > 80)
+    {
+        std::cout << "A" << std::endl;
+    }
```
Comment (Vishal-raj-1) [inline if-else questions/grade_card.cpp:32]: There should be last condition 
```
else if(marks > 100){ 
cout << "Invalid Marks";
}
```

**Review by Vishal-raj-1 — APPROVED** [review]
You are good going. I am merging your first PR !! Congratulations !! From next time make sure you are not writing std:: everywhere. 
Cheers

**Comment by Vishal-raj-1** [comment]
@AryaChakraborty You have to move the if-else folder into your named folder. Currently, it is outside.

**Comment by AryaChakraborty** [comment]
okay, I will do it now then.

**Pull request merged**